host-interaction/console/manipulate-console.yml

```diff
@@ -12,13 +13,18 @@ rule:
     examples:
       - 3aa7ee4d67f562933bc998f352b1f319:0x705413A0
   features:
-    - and:
+    - or:
+      - and:
+        - or:
+          - api: kernel32.SetConsoleCursorPosition
+          - api: kernel32.ReadConsoleOutputCharacter
+          - api: kernel32.WriteConsoleOutputCharacter
+          - api: kernel32.WriteConsoleOutput
+          - api: kernel32.WriteConsoleInput
+        - optional:
+          - api: kernel32.GetStdHandle
+          - number: 0xfffffff5 = STD_OUTPUT_HANDLE
       - or:
-        - api: kernel32.SetConsoleCursorPosition
-        - api: kernel32.ReadConsoleOutputCharacter
-        - api: kernel32.WriteConsoleOutputCharacter
-        - api: kernel32.WriteConsoleOutput
-        - api: kernel32.WriteConsoleInput
-      - optional:
-        - api: kernel32.GetStdHandle
-        - number: 0xfffffff5 = STD_OUTPUT_HANDLE
+        - api: System.Console::SetWindowSize
+        - api: System.Console::SetWindowPosition
```

Comment:
Do these capture the goal of this rule (I haven't checked the docs, but just based on their name)?

Reply:
`manipulate console`  is a fairly broad rule name that I'd argue includes the referenced APIs based on their MSDN descriptions:

```
Sets the height and width of the console window to the specified values.
Sets the position of the console window relative to the screen buffer.
```

Now if we tightened the rule name to something like `manipulate console output` the referenced APIs would no longer fit and we could instead split them to a new rule called something like `manipulate console window`.

I see the following options:
1. Leave as-is
2. Rename `manipulate console` to `manipulate console output` and create a second rule called `manipulate console window` that includes the referenced APIs

I favor option 2. Thoughts @mr-tz ?

Reply:
2. sounds good, for the first name I suggest `manipulate console buffer` (output and input).